addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -20,30 +20,58 @@ sAnimations.registry.ProductWishlist = sAnimations.Class.extend(ProductConfigura
     },
     events: sAnimations.Class.events,
 
-    start: function () {
+    /**
+     * Init variables.
+     *
+     * @override
```

Comment:
For constructors, no 'init variables' comment (useless), no `@override` (useless) but only a `@constructor`

Reply:
ok